From e80c23dabe0a5f48fdcdecfe4f1d65e73a46d591 Mon Sep 17 00:00:00 2001
From: AngeloGioacchino Del Regno <kholk11@gmail.com>
Date: Wed, 8 Dec 2021 14:26:10 +0200
Subject: [PATCH 004/175] msm: sde_hw_catalog: Exclude interface tear 1/2 irqs
 on

SDM845's MDP manages the tear interrupts inside the MDP interrupt
itself, so there's no external tear interrupt.
---
 techpack/display/msm/sde/sde_hw_catalog.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/techpack/display/msm/sde/sde_hw_catalog.c b/techpack/display/msm/sde/sde_hw_catalog.c
index 6a20a15c55d0..b9678f7f35ab 100644
--- a/techpack/display/msm/sde/sde_hw_catalog.c
+++ b/techpack/display/msm/sde/sde_hw_catalog.c
@@ -4246,6 +4246,8 @@ static int _sde_hardware_pre_caps(struct sde_mdss_cfg *sde_cfg, uint32_t hw_rev)
 		sde_cfg->ts_prefill_rev = 2;
 		sde_cfg->sui_misr_supported = true;
 		sde_cfg->sui_block_xin_mask = 0x3F71;
+		clear_bit(MDSS_INTF_TEAR_1_INTR, sde_cfg->mdss_irqs);
+		clear_bit(MDSS_INTF_TEAR_2_INTR, sde_cfg->mdss_irqs);
 		clear_bit(MDSS_INTR_LTM_0_INTR, sde_cfg->mdss_irqs);
 		clear_bit(MDSS_INTR_LTM_1_INTR, sde_cfg->mdss_irqs);
 		sde_cfg->has_decimation = true;
-- 
2.48.1

